testing/pac-ts-eks/tests/index.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
+
+import * as aws from "@pulumi/aws";
+import * as policy from "@pulumi/policy";
+import * as pulumi from "@pulumi/pulumi";
+
+const stackPolicy: policy.StackValidationPolicy = {
+    name: "eks-test",
+    description: "EKS integration tests.",
+    enforcementLevel: "mandatory",
+    validateStack: async (args, reportViolation) => {
+        const clusterResource = args.resources.find(r => r.isType(aws.eks.Cluster));
+        const cluster = clusterResource && clusterResource.asType(aws.eks.Cluster);
+        if (!cluster) {
+            reportViolation("EKS Cluster not found");
+            return;
+        }
+
+        if (cluster.version !== "1.13") {
+            reportViolation(`Expected EKS Cluster '${cluster.name}' version to be '1.13' but found '${cluster.version}'`);
+        }
+
+        const vpcId = cluster.vpcConfig.vpcId;
+        if (!vpcId) {
+            if (!pulumi.runtime.isDryRun()) {
```

Comment:
Might be worth adding a comment here why we check this so that example readers understand what's going on here.

Reply:
Done in #623 and applies to the guide